JS-Algorithms/challenges.js

```diff
@@ -96,12 +122,24 @@ Since 10! === 3628800 and you sum 3 + 6 + 2 + 8 + 8 + 0 + 0
 
 const digitSum = (n) => {
   // YOUR CODE HERE...
+  let x = 1;
+  for(let i = n; i >= 1; i--) {
+    x = x * i
+  }
+  const getLongNumber = BigInt(x);
```

Comment:
I would avoid variable names with 'get' prefixes. Seeming as this variable is only used in one place iI would say its not needed